Makefile.in

```diff
@@ -159,7 +160,7 @@ am__define_uniq_tagged_files = \
 ETAGS = etags
 CTAGS = ctags
 CSCOPE = cscope
-DIST_SUBDIRS = $(SUBDIRS)
+DIST_SUBDIRS = src lua
```

Comment:
@jacobkeeler why dont't you use variable SUBDIRS? You added `lua` to `SUBDIRS` above. didn't you?

Reply:
The file was generated by automake/autoconf, I don't know for certain why this was changed